src/app/systelab-components/chip-button/chip-button.component.ts

```diff
@@ -81,6 +90,9 @@ export class ChipButtonComponent {
 			this.buttonList.splice(index, 1);
 			this.buttonList = this.buttonList;
 		}
+		this.buttonRemoved.emit(item);
+		const last = this.buttonList[this.buttonList.length - 1];
+		this.selectItem(last);
```

Comment:
The last item must be selected only when the removed item was selected. Consider that the user can delete a chip button while another one is selected.